primitives/arithmetic/src/helpers_128bit.rs

```diff
@@ -75,6 +75,7 @@ mod double128 {
 	}
 
 	/// Returns 2^128 / a
+	/// `checked_div()` could be used here but is not possible to `unwrap()` inside a `const fn`
```

Comment:
I think you can use a `match` or `if` in const functions. 
  
But for this function i would probably make more sense to add a `checked_div128` function instead of changing it.